config/secrets.yml

```diff
@@ -24,6 +24,13 @@ test:
   ezid_password: <%= ENV['EZID_PASSWORD'] %>
   preservation_queue_name: <%= ENV['PMPY_QUEUE_NAME'] || 'test:pmpy_queue' %>
   rollbar_token: ''
+staging:
+  secret_key_base: 450fc6b88e8a27fe9ca23621599762b9faab14421121131d98f6f4de737cd8a74266fd7e2c0f7316a2388066722ea01e354bb9930f4ea223bd7bd02cf11e65c3
```

Comment:
should be the env var: `secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>`

You want to `rake secret` this to generate this token. Otherwise having this public opens a whole can of security worms